pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +314,25 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+// getLogWithStacktracing search for the first stack trace line and offset it by logLinesOffset
+func getLogWithStacktracing(logArray []string) string {
+	var log []string
+	var found bool
+
+	for idx := range logArray {
+		line := logArray[idx]
+		log = append(log, line)
+		if !found {
+			if len(log) > logLinesOffset {
+				log = log[1:]
```

Comment:
But doesn't it still store all the strings in the underlying array in memory?

Reply:
TBH I don't like the whole idea of implementing circular buffer like this and I don't think we need a circular buffer either, it'd be more clear to first find the string with the stack trace and then take all the necessary log lines from the original array. It'll be logLinesOffset more iterations, but I think not a big deal.

Reply:
@Sergey1011010 check it now.